packages/react-tabs/src/components/TabList/useTabListStyles.ts

```diff
@@ -1,11 +1,17 @@
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
-import type { TabListState } from './TabList.types';
+import type { TabListSlots, TabListState } from './TabList.types';
 import { tokens } from '@fluentui/react-theme';
-import { usePrevious } from '@fluentui/react-utilities';
+import { SlotClassNames, usePrevious } from '@fluentui/react-utilities';
 import { tabPendingDesignTokens } from '../../tab.constants';
 
+/**
+ * @deprecated
+ */
 export const tabListClassName = 'fui-TabList';
```

Comment:
For `TabList` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```